### ISSUE TITLE
Skip forward and back 15s Option

### ISSUE THREAD

**Issue by WiseSmil3** [FossifyOrg/Music-Player#86]
### Checklist

- [ ] I made sure that there are **no existing issues** - [open](https://github.com/FossifyOrg/Music-Player/issues) or [closed](https://github.com/FossifyOrg/Music-Player/issues?q=is%3Aissue+is%3Aclosed) - to which I could contribute my information.
- [x] I made sure that there are **no existing discussions** - [open](https://github.com/FossifyOrg/Music-Player/discussions) or [closed](https://github.com/FossifyOrg/Music-Player/discussions?discussions_q=is%3Aclosed) - to which I could contribute my information.
- [X] I have read the FAQs inside the app (Menu -> About -> FAQs) and my problem isn't listed.
- [X] **I have taken the time to fill in all the required details. I understand that the bug report will be dismissed otherwise.**
- [X] This issue contains only one feature request.
- [X] I have read and understood the [contribution guidelines](https://github.com/FossifyOrg/Music-Player/blob/master/CONTRIBUTING.md).
- [ ] I optionally [donated](https://fossify.org/donate) to support the Fossify mission.

### Feature description

Add a Skip forward and backwards buttons to the left and the right side of the play button.

### Why do you want this feature?

This will help very much when skipping forward and backwards in audio books. In long tracks the track pointer is hard to place precisely.

<img alt="movie_stream_backwards_15s-512-4060426797" src="https://github.com/user-attachments/assets/8037dca8-161c-4978-a0f8-70fc6fbbae61" width=400 />

**Comment by Aga-C** [comment]
It's already implemented. When you tap on time, it jumps 10 s.

**Comment by tbilles** [comment]
Hi, is there a way to skip backwards / forwards 10s on the lock screen widget? I couldn't find it, only the prev / next song buttons.

**Comment by JustMyGithub** [comment]
Just want to add +1 on wish for x seconds seek on notification widget and suggest a setting for number of seconds to seek.

btw: "click time to seek" is very unintuitive - I did not notice until I wanted to suggest the feature. There is enough space for seprate buttons, isn't there? I think the way it is many users do not even know the feature exists.

**Comment by naveensingh** [comment]
I agree, reopening for seek button in notifications but something must be done about the in-app buttons as well.
